File: CodingBat/Python/Warmup_1/missing_char.py
#Given a non-empty string and an int n, return a new string where the char at the index n has been removed. The value of n will be a valid index of a char in the origional string (i.e. n will be in the reange 0..len(str)-1 inclusive).
def missing_char(str,n):
    i=0
    x=""
    for l in str:
        if i==n:
            i+=1
        else:    
            x=x+str[i]
            i+=1
    return x

File: CodingBat/Java/Warmup_1/missingChar.java
public class missingChar{
    // Given a non-empty string and an int n, return a new string where the char at index n hs been removed. The value of n will be a valid index of a char in the original string
    public String missingChar(String str, int n){
        String x="";
        for (int i=0;i<str.length();i++){
            if(i!=n){
                char charAt = str.charAt(i);
                x+=charAt;
            }
          
        }
        return x;
    }
    public static void main(String[] args) {
        
    }
}